core/src/main/java/com/radixdlt/consensus/bft/VertexStoreJavaImpl.java

```diff
@@ -183,6 +183,7 @@ public Option<VertexStoreState> tryRebuild(VertexStoreState vertexStoreState) {
     this.rootVertex = vertexStoreState.getRoot();
     this.highestCommittedQC = vertexStoreState.getHighQC().highestCommittedQC();
     this.highestQC = vertexStoreState.getHighQC().highestQC();
+    this.highestTC = vertexStoreState.getHighQC().highestTC();
```

Comment:
Does it make sense to just store `highQC` on this class? instead of these separate values?